spring-cloud-gcp-pubsublite/pom.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.google.cloud</groupId>
+    <artifactId>spring-cloud-gcp</artifactId>
+    <version>2.0.3-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>spring-cloud-gcp-pubsublite</artifactId>
+  <name>Spring Cloud GCP Module - Pub/Sub Lite</name>
+  <description>Spring Cloud GCP Pub/Sub Lite Module</description>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-pubsublite</artifactId>
+      <version>0.13.0</version>
```

Comment:
Is Pubsub Lite included in the Google libraries BOM? If so, I think we'd prefer to use that version vs. hardcoding here.

Reply:
It is not currently.